torch/_dynamo/bytecode_transformation.py

```diff
@@ -57,6 +57,12 @@ def create_instruction(name, arg=None, argval=_NotProvided, target=None):
     )
 
 
+# Python 3.11 remaps
+def create_jump_absolute(target):
+    inst = "JUMP_FORWARD" if sys.version_info >= (3, 11) else "JUMP_ABSOLUTE"
+    return create_instruction(inst, target=target)
```

Comment:
I'm surprised the target is the same for the both versions while one is an absolute value while the other is supposed to be a relative offset?

Reply:
Inductor has a python byte code assembler lower down that converts object pointers into offsets.  We should make sure it knows how to generate JUMP_BACKWARD though.